src/IB/IBFEMethod.cpp

```diff
@@ -944,6 +1020,13 @@ IBFEMethod::spreadForce(const int f_data_idx,
                                                 /*close_X*/ false);
     }
 
+    if (d_use_scratch_hierarchy)
+    {
+        // TODO: this assumes the structure is always on the finest level
+        // TODO: we don't need a RefinePatchStrategy here, right?
```

Comment:
If it is just a straightforward copy, then no.

This could be a good place to do the accumulation from the ghost region into interior, however, and could use a patch strategy to accomplish that.

Reply:
This would require a significant change to the way we do this: we currently accumulate inside `FEDataManager`. It would be a lot more efficient to do the
1. setup temporary force data index
2. spread force
3. accumulate force
4. add temporary force to given index

inside IBFEMethod but this changes behavior.

Let me open an issue for this so that we can address it later: I would prefer to leave it out of this PR since it is only tangentially related to the scratch hierarchy.